File: map-api/src/main/kotlin/domain/OperatingPeriod.kt
package cz.cvut.fit.gaierda1.domain

import java.time.LocalDateTime
import java.time.ZoneId

data class OperatingPeriod(
    val timeZone: ZoneId,
    val fromDate: LocalDateTime,
    val toDate: LocalDateTime,
    val validDays: List<Boolean>,
){
}

File: map/src/api/model/operatingPeriod.ts
type OperatingPeriod = {
    timezone: string,
    fromDate: string,
    toDate: string,
    validDays: string,
}

type OperatingPeriodWithDates = {
    fromDate: Date,
    toDate: Date,
    validDays: string,
}

export type { OperatingPeriod, OperatingPeriodWithDates }
